### PR TITLE
right align row-counter

### PR DESCRIPTION
**Pull request by minusf**

### CODE REVIEW

File: controlcenter/static/controlcenter/css/all.css [hunk 1]
```diff
@@ -43,7 +44,6 @@
   background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAABGCAAAAAAoI6GBAAAAOElEQVR4ATWNhQEAMAjDoP/fO3cJkwrmTU1dprmfX/TLwfefA6J2xWEfh4wJ1NJLLTtuNbuYEpUWQ5QZZ928F4cAAAAASUVORK5CYII="), url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAAAAADFHGIkAAAAQElEQVR4AXXIOQHAAACEMPzbxMEp6NafjMG7XYiXeImXeImXeImXeImXeImXeImX+BE/4kf8iB/xI37Ej/gRvwPQEA5XmMlaIAAAAABJRU5ErkJggg==");
 }
 
-.controlcenter__widget__subtitle,
 .controlcenter__widget__tab {
```
Comment (byashimov) [inline controlcenter/static/controlcenter/css/all.css]: Why did you dropped this? It is actually [exists](https://github.com/byashimov/django-controlcenter/blob/71dea21ab0b9f1693c923f3fa824c5436eb850eb/controlcenter/templates/controlcenter/dashboard.html#L31).

File: controlcenter/static/controlcenter/css/all.css [hunk 2]
```diff
@@ -213,12 +218,14 @@
...
 .controlcenter__table[data-sortable] .controlcenter__table__thead .controlcenter__table__th[data-sorted-direction='ascending'],.controlcenter__table[data-sortable] .controlcenter__table__thead .controlcenter__table__th[data-sorted-direction='descending'] {
+  background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(0,0,0,0)), to(rgba(0,0,0,0.035)));
   background-image: linear-gradient(to bottom, rgba(0,0,0,0) 0%, rgba(0,0,0,0.035) 100%);
   color: #333 !important;
 }
 
 .controlcenter__table[data-sortable] .controlcenter__table__thead .controlcenter__table__th[data-sorted-direction='descending'] {
   color: #333 !important;
+  background-image: -webkit-gradient(linear, left bottom, left top, from(rgba(0,0,0,0)), to(rgba(0,0,0,0.05)));
   background-image: linear-gradient(to top, rgba(0,0,0,0) 0%, rgba(0,0,0,0.05) 100%);
```
Comment (byashimov) [inline controlcenter/static/controlcenter/css/all.css]: Did grunt made this? webkit prefix is for very old browsers.

File: controlcenter/static/controlcenter/css/all.css [hunk 3]
```diff
@@ -308,15 +315,15 @@
...
   }
 
   .controlcenter__masonry__block--w3 {
     width: 50%;
   }
 
   .controlcenter__masonry__block--w4 {
-    width: 66.66666666666667%;
+    width: 66.66666666666666%;
   }
```
Comment (byashimov) [inline controlcenter/static/controlcenter/css/all.css]: Which browser di you use? Have you checked other browsers then? This can brake things.

**Comment by minusf** [comment]
i am sorry if i was not clear in my other comment.  you can see my only true change in the `all.styl` file, all the other changes were the result of me running `grunt`.  i installed grunt using `npm i` which used your package definitions.  if the grunt task generates different style files when run by different people, perhaps there is something wrong with this setup...

File: controlcenter/static/controlcenter/css/all.css (hunk 1 above)
Comment (byashimov) [reply controlcenter/static/controlcenter/css/all.css]: Looks like it was added in css directly. Should be added to styl and regenerate css.

**Comment by byashimov** [comment]
Hi, this one has conflicts.
And if you have time, could you add `.controlcenter__widget__subtitle,` to styl file, so if won't be deleted from css? Thanks.

**Pull request merged**